fff

```diff
@@ -1116,10 +1159,18 @@ main() {
     mkdir -p "${XDG_CACHE_HOME:=${HOME}/.cache}/fff" \
              "${FFF_TRASH:=${XDG_DATA_HOME:=${HOME}/.local/share}/fff/trash}"
 
+    USER=$(whoami)
```

Comment:
Isn't `$USER` defined by default? If not guaranteed, use a variable with fallback like `user="${USER:-"$(whoami)"}"`